doc/index.txt

```diff
@@ -21,6 +21,9 @@ learning/machine learning <https://mila.umontreal.ca/en/cours/>`_ classes).
 News
 ====
 
+* Removed support for the old (device=gpu) backend.  Use the new
+  backend (device=cuda) for gpu computing.
+
```

Comment:
This is the doc for the dev version.  We are not on the release branch.  I'll add the link.